HomeUI/src/views/apps/Management.vue

```diff
@@ -5518,10 +5843,7 @@ export default {
         totalRows: 1,
         currentPage: 1,
       },
-      total: '',
-      downloaded: '',
       downloadedSize: '',
-      abortToken: {},
```

Comment:
this 3 deleted properties are no longer used?

Reply:
duplication